tests/test/faaslet/test_pool.cpp

```diff
@@ -230,56 +224,4 @@ TEST_CASE("Test pool accounting", "[faaslet]")
     w1.bindToFunction(call);
     REQUIRE(pool.getThreadCount() == 2);
 }
-
-TEST_CASE("Test worker lifecycle interacts with scheduler", "[faaslet]")
```

Comment:
This is basic functionality associated with Faabric executors, and tested in Faabric itself.